mpi.h

```diff
@@ -335,24 +335,24 @@ extern int MPI_Group_intersection(MPI_Group group1, MPI_Group group2,
 extern int MPI_Group_difference(MPI_Group group1, MPI_Group group2,
                          MPI_Group *newgroup);
 extern int MPI_Group_free(MPI_Group *group);
-extern int MPI_Group_translate_ranks(MPI_Group group1, int n, int *ranks1,
+extern int MPI_Group_translate_ranks(MPI_Group group1, int n, const int ranks1[],
                                      MPI_Group group2, int *ranks2);
 extern MPI_Group MPI_Group_f2c(MPI_Fint group);
 extern MPI_Fint MPI_Group_c2f(MPI_Group group);
 
-extern int MPI_Init(int *argc, char **argv[]) ;
+extern int MPI_Init(int *argc, char ***argv) ;
```

Comment:
MPI serial library does not have to have the same decl (argv[] vs *argv) as MPICH, although not an issue IMO

Reply:
Yes, I just want to make it exactly the same as MPICH in this PR, even though this change is not strictly necessary.